programs/Makefile

```diff
@@ -143,219 +150,219 @@ psa/psa_constant_names$(EXEXT): psa/psa_constant_names_generated.c
 psa/psa_constant_names_generated.c: ../scripts/generate_psa_constants.py ../include/psa/crypto_values.h ../include/psa/crypto_extra.h
 	../scripts/generate_psa_constants.py
 
-aes/aescrypt2$(EXEXT): aes/aescrypt2.c $(DEP)
+aes/aescrypt2$(EXEXT): aes/aescrypt2.c $(MBEDLIBS) mbedtls_test
```

Comment:
Instead of repeating `$(MBEDLIBS) mbedtls_test` (or, as suggested above, `$(MBEDLIBS) $(MBEDTLS_TEST_OBJS)`), why not define `DEP=$(MBEDLIBS) $(MBEDTLS_TEST_OBJS)` and keep `$(DEP)` here?

Reply:
Indeed, thanks.